src/main.rs

```diff
@@ -53,8 +53,17 @@ fn main() {
                 .short("e")
                 .takes_value(true)
                 .possible_values(&["hyper", "reqwest"])
+                .default_value("hyper")
                 .help("The engine to use"),
         )
+        .arg(
+            Arg::with_name("header")
+                .long("header")
+                .multiple(true)
+                .takes_value(true)
+                .number_of_values(1)
+                .help("Headers to inject in the request. Example '--header user-agent=rust-rench'"),
```

Comment:
Can we use a different delimiter other than `=`? The equals sign is part of cli options parsing and I think it could make for a confusing input string. Perhaps a colon?

```bash
--header user-agent:rust-rench
```

Reply:
+1 for `:`, lines up with curl which is nice